lib/remap-functions/src/includes.rs

```diff
@@ -0,0 +1,166 @@
+use remap::prelude::*;
+
+#[derive(Clone, Copy, Debug)]
+pub struct Includes;
+
+impl Function for Includes {
+    fn identifier(&self) -> &'static str {
+        "includes"
+    }
+
+    fn parameters(&self) -> &'static [Parameter] {
+        &[
+            Parameter {
+                keyword: "value",
+                accepts: |v| matches!(v, Value::Array(_)),
+                required: true,
+            },
+            Parameter {
+                keyword: "item",
+                accepts: |_| true,
+                required: true,
+            },
+        ]
+    }
+
+    fn compile(&self, mut arguments: ArgumentList) -> Result<Box<dyn Expression>> {
+        let value = arguments.required("value")?.boxed();
+        let item = arguments.required("item")?.boxed();
+
+        Ok(Box::new(IncludesFn { value, item }))
+    }
+}
+
+#[derive(Debug, Clone)]
+struct IncludesFn {
+    value: Box<dyn Expression>,
+    item: Box<dyn Expression>,
+}
+
+impl Expression for IncludesFn {
+    fn execute(&self, state: &mut state::Program, object: &mut dyn Object) -> Result<Value> {
+        let list = self.value.execute(state, object)?.try_array()?;
+
+        let included = if list.is_empty() {
+            false
+        } else {
+            let item = self.item.execute(state, object)?;
+            list.iter().any(|i| i == &item)
+        };
```

Comment:
This might be a bit goofy, but it's more concise:
```suggestion
        let included = !list.is_empty() && {
            let item = self.item.execute(state, object)?;
            list.iter().any(|i| i == &item)
        };
```